docs/UserGuide.md

```diff
@@ -22,11 +22,11 @@ Understand more about E-Duke-8, and how you can use it to aid your learning, thr
 ## 2. Quick Start
 
 1. Ensure that you have Java 11 or above installed.
-2. Download the latest version of **E-Duke-8** from [here](https://github.com/AY2021S1-CS2113T-F12-3/tp/releases).
+2. Download the latest version of the jar file for **E-Duke-8** from [here](https://github.com/AY2021S1-CS2113T-F12-3/tp/releases).
 3. Copy the files to the folder you want to use for the application.
-4. Double click on `eduke8.jar` to start up the program. If the program does not start up, continue to step 6, otherwise skip to step 8.
+4. Double click on the jar file to start up the program. If the program does not start up, continue to step 6, otherwise skip to step 8.
 5. If double clicking on the jar file does not work, open up the command prompt in the same folder used in step 3.
-6. Enter the following command `java -jar eduke8.jar`.
+6. Enter the following command `java -jar eduke8.jar` (assuming the downloaded jar file is named `eduke8.jar` but this can be changed).
```

Comment:
Looks great! It's good that we are more specific on these instructions, so as to avoid confusion with users